### PR TITLE
Set up Formideploy for Docusarus site

### PR DESCRIPTION
**Pull request by carloskelly13**
Sets up the pipeline to deploy our docs site to `formidable.com/open-source/react-live`

### CODE REVIEW

File: docs/introduction.mdx [hunk 1]
```diff
@@ -33,9 +35,12 @@ export const jsxExample = `
 To render a series of components or render components beyond just JSX, React Live also provides a `render` function to pass JSX into when the `noInline` prop is present. This lets you render multiple or functional components with hooks. This example shows a functional component with a `useState` hook.
 
 export const noInlineExample = `
-const Counter = () => {
+type Props = {
```
Comment (carloskelly13) [inline docs/introduction.mdx:38]: This is just adding Typescript-based sample code to our docs.

File: website/formideploy.config.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  lander: {
+    name: "react-live",
+  },
+  build: {
+    // Build output directory.
+    // Docusaurus defaults to `build`.
+    // We use `docusaurus build --out-dir build/open-source/docusaurus` to build intermediate real dirs.
+    dir: "build",
```
Comment (gksander) [inline website/formideploy.config.js:9]: I think in package.json, for the build script, you'll want to update the output dir. And then also check the baseUrl in docusaurus config to make sure it matches. Groqd should have an example of those two things (reviewing from mobile right now)

**Review by gksander — COMMENTED** [review]
Looks good, just want to verify we've got the build output and baseUrl configured properly

File: website/formideploy.config.js (hunk 2 above)
Comment (carloskelly13) [reply website/formideploy.config.js:9]: Thank you for catching that. Yeah, the paths were off. I believe I pushed the correct URLs now that mirror our configuration for `groqd`.

**Review by ryan-roemer — APPROVED** [review]
LGTM!

**Review by gksander — APPROVED** [review]
⛹️‍♂️

**Pull request merged**